seeddata.json

```diff
@@ -38,5 +38,13 @@
     "code": "",
     "name": "Adjusted Budget"
   }
+},
+{
+  "model": "infrastructure.financialyear",
+  "pk": 1,
+  "fields": {
+    "budget_year": "2019/2020",
+    "active": "True"
```

Comment:
this PR should mean that we can remove the active property, right?

Reply:
Yes, will remove that from seed data